src/NuGet.Indexing/ILoader.cs

```diff
@@ -16,5 +16,7 @@ public interface ILoader
         /// <param name="config">Configuration to use.</param>
         /// <returns>Returns true if the data has changed.</returns>
         bool Reload(IndexingConfiguration config);
+
+        DateTime? GetLastUpdateTime(string name);
```

Comment:
This change is relevant to `StorageLoader`, here I think the `name` is appropriate. In `FileLoader` it could be `fileName`